snuba/cli/devserver.py

```diff
@@ -152,6 +152,19 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                     "--consumer-group=metrics_group",
                 ],
             ),
+            (
+                "metrics-aggregates-consumer",
+                [
+                    "snuba",
+                    "multistorage-consumer",
+                    "--storage=metrics_distributions",
+                    "--storage=metrics_counters",
+                    "--storage=metrics_sets",
+                    "--auto-offset-reset=latest",
+                    "--log-level=debug",
+                    "--consumer-group=metrics_aggregates_group",
+                ],
+            ),
```

Comment:
I think we should be duplicating data this way as two consumers would be consuming the same topic and write on the same tables (via matview for the first one). 
Feel free to swap the old one with the new one.
Also you will want to use the same consumer group. So you pick up from the last committed offset.

Reply:
am I correct that your first sentence should read "we should not be"?

Reply:
"I think we would be duplicating data this way ..."

Reply:
I thought we were trying to keep the buckets table writes as a diagnostic thing for a while. Was I wrong on that?